### ISSUE TITLE
Unusual OSGi manifest entry

### ISSUE THREAD

**Issue by tpolecat** [milessabin/shapeless#510]
The `Import-Package` manifest entry includes the unusual entry `scala.quasiquotes;version="${range;[==,=+);${@}}"` which confuses Felix 4.2.1 ... it's certainly not a valid version range.

```
bundle (develop *)$ unzip -p shapeless_2.10-2.2.5.jar META-INF/MANIFEST.MF
Manifest-Version: 1.0
Bnd-LastModified: 1437587749607
Bundle-ManifestVersion: 2
Bundle-Name: com.chuusai.core
Bundle-SymbolicName: com.chuusai.core
Bundle-Version: 2.2.5
Created-By: 1.8.0_51 (Oracle Corporation)
Export-Package: shapeless;version="2.2.5";uses:="scala,scala.annotation,
 scala.collection,scala.collection.generic,scala.collection.immutable,sc
 ala.reflect,scala.reflect.api,scala.reflect.macros,scala.runtime,scala.
 tools.nsc,scala.tools.nsc.typechecker,scala.util,shapeless.ops,shapeles
 s.ops.record,shapeless.syntax",shapeless.ops;version="2.2.5";uses:="sca
 la,scala.collection,scala.collection.generic,scala.collection.immutable
 ,scala.collection.mutable,scala.math,scala.reflect,scala.runtime,scala.
 util,shapeless,shapeless.ops.record",shapeless.ops.record;version="2.2.
 5";uses:="scala,scala.collection.immutable,scala.reflect,scala.runtime,
 shapeless",shapeless.syntax;version="2.2.5";uses:="scala,scala.collecti
 on,scala.collection.immutable,scala.reflect,scala.runtime,scala.util,sh
 apeless,shapeless.ops,shapeless.ops.record",shapeless.syntax.std;versio
 n="2.2.5";uses:="scala,scala.collection,scala.collection.immutable,scal
 a.reflect,scala.runtime,shapeless,shapeless.ops,shapeless.syntax",shape
 less.test;version="2.2.5";uses:="scala,scala.reflect,scala.reflect.api,
 scala.reflect.macros,scala.runtime"
Import-Package: scala;version="[2.10,2.11)",scala.annotation;version="[2
 .10,2.11)",scala.collection;version="[2.10,2.11)",scala.collection.gene
 ric;version="[2.10,2.11)",scala.collection.immutable;version="[2.10,2.1
 1)",scala.collection.mutable;version="[2.10,2.11)",scala.math;version="
 [2.10,2.11)",scala.quasiquotes;version="${range;[==,=+);${@}}",scala.re  <---- hmm
 flect;version="[2.10,2.11)",scala.reflect.api;version="[2.10,2.11)",sca
 la.reflect.internal;version="[2.10,2.11)",scala.reflect.macros;version=
 "[2.10,2.11)",scala.reflect.macros.runtime;version="[2.10,2.11)",scala.
 reflect.runtime;version="[2.10,2.11)",scala.runtime;version="[2.10,2.11
 )",scala.sys;version="[2.10,2.11)",scala.tools.nsc;version="[2.10,2.11)
 ",scala.tools.nsc.typechecker;version="[2.10,2.11)",scala.util;version=
 "[2.10,2.11)"
Tool: Bnd-2.1.0.20130426-122213
```

**Comment by milessabin** [comment]
Tagging @mpilquist ...

**Comment by tpolecat** [comment]
Package `scala.quasiquotes` is not exported by any OSGi bundle so I'm not sure what BND is supposed to do here. There's a version in the manifest but it's not really meaningful in this context.

**Comment by tpolecat** [comment]
Note also

```
Bundle-Name: com.chuusai.core
Bundle-SymbolicName: com.chuusai.core
```

**Comment by mpilquist** [comment]
I forgot about this one, sorry!

**Comment by milessabin** [comment]
I'm aiming for an RC by the end of this week ... any insight on this one?

**Comment by milessabin** [comment]
Fixed in #540.
